plugins/inventory/proxmox.py

```diff
@@ -27,17 +27,29 @@
         choices: ['community.general.proxmox']
         type: str
       url:
-        description: URL to Proxmox cluster.
+        description:
+          - URL to Proxmox cluster.
+          - If the value is not specified in the task, the value of environment variable C(PROXMOX_URL) will be used instead.
```

Comment:
This sentence is not necessary. If you prefer to keep it, that's also ok though. If you want to keep it, you need to change it though, since inventory plugins do not specify parameters in tasks, but in inventory files:
```suggestion
          - If the value is not specified in the inventory configuration, the value of environment variable C(PROXMOX_URL) will be used instead.
```
(same for the others).

Reply:
Addressed in https://github.com/ansible-collections/community.general/pull/1645/commits/a111f6a34146162349b691bfda9269e30ee660bd